beacon_node/http_api/src/lib.rs

```diff
@@ -1295,9 +1300,9 @@ pub fn serve<T: BeaconChainTypes>(
                         &network_tx,
                         log,
                         BroadcastValidation::default(),
+                        duplicate_block_status_code,
```

Comment:
Just for my understanding - the issue reported is only relevant to the blinded flow right? The change made to the non blinded flows are just for consistency I'm guessing?

Reply:
One of the relays also hit the error with full blocks when broadcasting to multiple BNs, and it can happen for VCs that broadcast too (e.g. Vouch).